pkg/resourceinterpreter/default/thirdparty/resourcecustomizations/batch.volcano.sh/v1alpha1/Job/customizations.yaml

```diff
@@ -0,0 +1,251 @@
+apiVersion: config.karmada.io/v1alpha1
+kind: ResourceInterpreterCustomization
+metadata:
+  name: declarative-configuration-job
+spec:
+  target:
+    apiVersion: batch.volcano.sh/v1alpha1
+    kind: Job
+  customizations:
+    replicaRevision:
+      luaScript: >
+        function ReviseReplica(obj, replica)
+          obj.spec.minAvailable = replica
+          return obj
+        end
+    healthInterpretation:
+      luaScript: >
+        function InterpretHealth(observedObj)
+          if observedObj.status == nil then
+            return false
+          end
+          if observedObj.status.state.phase == nil or observedObj.status.state.phase == '' or observedObj.status.state.phase == 'Failed' or observedObj.status.state.phase == 'Terminated' or observedObj.status.state.phase == 'Terminating' then
+            return false
+          end
+          return true
+        end
+    componentResource:
+      luaScript: |
+        local kube = require("kube")
+        local function isempty(s)
+          return s == nil or s == ''
+        end
+
+        local function get(obj, path)
+          local cur = obj
+          for i = 1, #path do
+            if cur == nil then return nil end
+            cur = cur[path[i]]
+          end
+          return cur
+        end
+
+        local function clone_plain(val, seen)
+          local tv = type(val)
+          if tv ~= "table" then return val end
+          seen = seen or {}
+          if seen[val] then return nil end
+          seen[val] = true
+          local out = {}
+          for k, v in pairs(val) do
+            local tk = type(k)
+            if tk == "string" or tk == "number" then
+              local cv = clone_plain(v, seen)
+              if cv ~= nil then out[k] = cv end
+            end
+          end
+          seen[val] = nil
+          return out
+        end
+
+        local function apply_pod_template(pt_spec, requires)
+          if pt_spec == nil then return end
+          local nodeSelector = clone_plain(pt_spec.nodeSelector)
+          local tolerations  = clone_plain(pt_spec.tolerations)
+          local priority     = pt_spec.priorityClassName
+
+          if nodeSelector ~= nil or tolerations ~= nil then
+            requires.nodeClaim = requires.nodeClaim or {}
+            requires.nodeClaim.nodeSelector = nodeSelector
+            requires.nodeClaim.tolerations  = tolerations
+          end
+
+          if not isempty(priority) then
+            requires.priorityClassName = priority
+          end
+        end
```

Comment:
Does the `kube.accuratePodRequirements` simplify the pod template parsing?
Like https://github.com/karmada-io/karmada/blob/13c28a00b21cd0afa9f05962ae8c5ad831d3e1f0/pkg/resourceinterpreter/default/thirdparty/resourcecustomizations/argoproj.io/v1alpha1/Workflow/customizations.yaml#L23